setup.py

```diff
@@ -78,7 +78,7 @@
         'graphviz>=0.8',
         'networkx>=2.0.0',
         'six>=1.10.0',
-        'torch',
+        'torch>=0.4.0',
```

Comment:
If we also add `numpy` to `install_requires`, then we can just do a `pip install pyro-ppl` out of the box.

Also, this one liner will only work on mac I think, as the binaries for linux/windows aren't hosted on pypi.